src/main/java/seedu/planner/model/ModulePlanner.java

```diff
@@ -13,9 +13,10 @@
  */
 public class ModulePlanner {
 
-    private final int maxSemesters = 8;
+    public static final int MAX_NUMBER_SEMESTERS = 8;
+    public static final int NUMBER_TYPES_USER_MODULES = 2;
```

Comment:
It is not clear what this constant means and it is used. Can you explain? You might want to add a comment there too.

Reply:
Ok to the post on JsonModuleInfoStorage